spec/index.md

```diff
@@ -2053,6 +2061,8 @@ Indicates various information about the canister. It contains:
 
 -   The cycle balance of the canister.
 
+-   The canister log visibility of the canister.
```

Comment:
You  specified the canister log visibility as part of `definite_canister_settings` so it'll be returned here.

Reply:
Ack.